src/store/sagas/positions.ts

```diff
@@ -143,9 +143,19 @@ function* handleInitPosition(action: PayloadAction<InitPositionData>): Generator
       })
     )
 
-    const signedBatchedTx = yield* call([batchedTx, batchedTx.signAsync], walletAddress, {
-      signer: adapter.signer as Signer
-    })
+    let signedBatchedTx = null
+
+    try {
+      signedBatchedTx = yield* call([batchedTx, batchedTx.signAsync], walletAddress, {
+        signer: adapter.signer as Signer
+      })
+    } catch (e) {
+      throw new Error('Error while signing transaction')
```

Comment:
this could be the function like: `throwSigningError`,  or you can add at least the error text: 'Error while signing transaction' as a const because it is repeated a lot of times.